modules/virt-delete-vm-web.adoc

```diff
@@ -1,22 +1,33 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virt-delete-vms.adoc
+// * virt/managing-vms/virt-delete-vms.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-delete-vm-web_{context}"]
 
 = Deleting a virtual machine using the web console
 
-Deleting a virtual machine permanently removes it from the cluster.
+Deleting a virtual machine (VM) permanently removes it from the cluster.
+
+If the VM is delete protected, the *Delete* action is disabled in the VM's *Actions* menu. To delete the VM you must first disable the VM's delete protection setting.
+
+
 
 .Procedure
 
-. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+. From the {product-title} web console, choose your view:
 
-. Click the Options menu {kebab} beside a virtual machine and select *Delete*.
+    * For a virtualization-focused view, select *Administrator* → *Virtualization* → *VirtualMachines*.
+
+    * For a general view, navigate to *Virtualization* → *VirtualMachines*.
+
+. Click the *Actions* menu {kebab} beside a VM and select *Delete*.
 +
-Alternatively, click the virtual machine name to open the *VirtualMachine details* page and click *Actions* -> *Delete*.
+Alternatively, click the VM's name to open the *VirtualMachine details* page and click *Actions* -> *Delete*.
```

Comment:
I know it was already here, but same thing about if we can remove the alternative method and just show one way in the procedure, to reduce complexity.